CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+## Waxu Chef v4.0.4_1.11.0
+
+### Added
+- Bump version 4.0.4
```

Comment:
Please, follow the convention for this lines `Update to Wazuh version 4.0.x`, also, could you link the wazuh core changelog? https://github.com/wazuh/wazuh/blob/v4.0.4/CHANGELOG.md#v404